src/backend/serial/u32/constants.rs

```diff
@@ -234,11 +237,13 @@ pub const EIGHT_TORSION_INNER_DOC_HIDDEN: [EdwardsPoint; 8] = [
 ];
 
 /// Table containing precomputed multiples of the Ed25519 basepoint \\(B = (x, 4/5)\\).
+#[cfg(feature = "basepoint-tables")]
 pub static ED25519_BASEPOINT_TABLE: &'static EdwardsBasepointTable =
```

Comment:
Should this even be `pub` anymore? Now that there's a `mul_base()` method, I think it would be best if this constant gets hidden away. This would also remove the need for the `traits::BasepointTable` trait.